test/util.go

```diff
@@ -26,6 +33,95 @@ func WaitForMesh() {
 	time.Sleep(2 * time.Second)
 }
 
+// WaitForMeshWithMessage sets up a gossipsub network and sends a test message.
+// Blocks until all other hosts see the first host's message.
+func WaitForMeshWithMessage(t *testing.T, topic string, hosts ...host.Host) []*pubsub.Topic {
```

Comment:
Is it necessary for this function to create the slice of `pubsub.Topic`?  Wouldn't it be sufficient to just wait until message massing on the `startupCheck` topic was working?

Reply:
yes.

What you suggest is what I tried initially, but it didn't work. It's because the `pubsub.Topic` comes from the `pubsub.PubSub` object, so in order for the topic to get the properties we want, we need to create it from this `pubsub.PubSub`.